pkg/wifi/wifiStub/wifiStub.go

```diff
@@ -0,0 +1,25 @@
+package wifiStub
+
+import "github.com/u-root/u-root/pkg/wifi"
+
+type StubWifiWorker struct {
+	ScanInterfacesOut  []string
+	ScanWifiOut        []wifi.WifiOption
+	ScanCurrentWifiOut string
+}
+
+func (stub StubWifiWorker) ScanInterfaces() ([]string, error) {
+	return stub.ScanInterfacesOut, nil
+}
+
+func (stub StubWifiWorker) ScanWifi() ([]wifi.WifiOption, error) {
+	return stub.ScanWifiOut, nil
+}
+
+func (stub StubWifiWorker) ScanCurrentWifi() (string, error) {
+	return stub.ScanCurrentWifiOut, nil
+}
+
+func (StubWifiWorker) Connect(a ...string) error {
+	return nil
+}
```

Comment:
I'm going to assume the elements in StubWifiWorker are never set, which means you ought to be able to do this:
type NullWifi int
func (NullWifi) ScanInterfaces ([]string, error) {
return []string{}, nil
}

and so on
Then when you use this you can do something like
var v wifi.NullWifi
and use v.ScanInterfaces

and so on

and go ahead and not have this a separate package.